### PR TITLE
PAYEE-1004 - Addressing platops pr bot comments

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by sergi-hmrc — APPROVED** [review]

**Review by masahmad — APPROVED** [review]

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -33,7 +31,7 @@ private object AppDependencies {
     def apply() =
       new TestDependencies {
         override lazy val test = Seq(
-          "org.scalatest"          %% "scalatest"          % "3.0.9",
+          "org.scalatest"          %% "scalatest"          % "3.0.9" % scope,
```
Comment (PerrettJ4) [inline project/AppDependencies.scala:34]: What does the % Scope add?

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -15,8 +15,6 @@ private object AppDependencies {
     "uk.gov.hmrc"       %% s"bootstrap-frontend-$playVersion" % "5.3.0",
     "uk.gov.hmrc"       %% "domain"                           % s"8.1.0-$playVersion",
     "uk.gov.hmrc"       %% "url-builder"                      % s"3.7.0-$playVersion",
-    "uk.gov.hmrc"       %% "play-partials"                    % s"8.3.0-$playVersion",
-    "uk.gov.hmrc"       %% "play-language"                    % s"5.3.0-$playVersion",
```
Comment (PerrettJ4) [inline project/AppDependencies.scala:19]: How come these are now redundant?

File: project/plugins.sbt [hunk 3]
```diff
@@ -4,15 +4,15 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
```
Comment (PerrettJ4) [inline project/plugins.sbt:7]: In the PR-Bot it says 2.8.18 which obviously makes no sense but thought I'd mention

**Review by PerrettJ4 — APPROVED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (brendanstraughan) [reply project/AppDependencies.scala:34]: not sure, the bot said to do it so I done it

File: project/plugins.sbt (hunk 3 above)
Comment (brendanstraughan) [reply project/plugins.sbt]: yeah for some reason when I updated everything and pushed it then changed and told me to update everything further

File: project/AppDependencies.scala (hunk 1 above)
Comment (PerrettJ4) [reply project/AppDependencies.scala:34]: hahha yeah makes sense

**Review by vivekhmrc — APPROVED** [review]

**Pull request merged**